oem/ibm/service_files/pldm-create-phyp-nvram.service

```diff
@@ -4,12 +4,29 @@ Wants=obmc-vpnor-updatesymlinks.service
 After=obmc-vpnor-updatesymlinks.service
 Wants=obmc-flash-bios-init.service
 After=obmc-flash-bios-init.service
-ConditionPathExists=!/var/lib/phosphor-software-manager/hostfw/nvram/PHYP-NVRAM
 
 [Service]
-ExecStart=/usr/bin/create-NVRAM-file
 Type=oneshot
 RemainAfterExit=no
+ExecStart=/bin/sh -c '\
+  mkdir -p /var/lib/phosphor-software-manager/hostfw/nvram; \
+  if [ ! -f /var/lib/phosphor-software-manager/hostfw/nvram/PHYP-NVRAM ]; then \
+    NEED_CREATE=true; \
+  elif [ "$(stat -c%s /var/lib/phosphor-software-manager/hostfw/nvram/PHYP-NVRAM)" -ne $((1024 * 145408)) ]; then \
```

Comment:
For better readability can we add variables for NVRAM file?
like NVRAM_FILE="/var/lib/phosphor-software-manager/hostfw/nvram/PHYP-NVRAM"